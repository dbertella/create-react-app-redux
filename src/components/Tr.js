import React, { PropTypes } from 'react';

const Tr = (props) => {
  const { info } = props;
  return (
    <tr>
      {
        info.map((artistInfo, i) => <td key={i}>{artistInfo}</td>)
      }
    </tr>
  );
};

Tr.propTypes = {
  info: PropTypes.array.isRequired,
}

export default Tr;
