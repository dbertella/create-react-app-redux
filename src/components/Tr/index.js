import React, { PropTypes } from 'react';

const Tr = (props) => {
  const { info } = props;
  return (
    <tr>
      {
        info.map((text, i) => <td key={i}>{text}</td>)
      }
    </tr>
  );
};

Tr.propTypes = {
  info: PropTypes.array.isRequired,
};

export default Tr;
