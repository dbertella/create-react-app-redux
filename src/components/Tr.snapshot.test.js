import React from 'react';
import Tr from './Tr';
import renderer from 'react-test-renderer';

it('changes the class when hovered', () => {
  const component = renderer.create(
    <Tr info={[]} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
