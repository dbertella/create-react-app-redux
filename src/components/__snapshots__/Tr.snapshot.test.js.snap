exports[`test changes the class when hovered 1`] = `<tr />`;
